sequelize/sequelize.d.ts

```diff
@@ -3116,24 +3116,29 @@ declare module "sequelize" {
         }
 
         /**
+         * Shortcut for types used in FindOptions.attributes
+         */
+		type FindOptionsAttriburesArray = Array<string | [string, string] | fn | [fn, string] | cast | [cast, string]>;
+
+        /**
          * Options that are passed to any model creating a SELECT query
          *
          * A hash of options to describe the scope of the search
          */
-        interface FindOptions {
+		interface FindOptions {
```

Comment:
You've inserted tabs instead of spaces here. For continuity I'd suggest you use spaces like the rest of the file. Same on L3121.